src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml

```diff
@@ -81,8 +81,8 @@
                             </Image.Style>
                         </Image>
                         </Button>
-                        <Button Name="CloseButton" Margin="0,7,5,7" BorderThickness="0"  Background="{StaticResource autocompletionWindow}">
-                            <Image Width="14" Height="14" HorizontalAlignment="Right" VerticalAlignment="Center">
+                        <Button Name="CloseButton" Margin="0,7,5,7" Height="14" Width="14" BorderThickness="0" Click="CloseAutocompletionWindow" Background="{StaticResource autocompletionWindow}">
```

Comment:
It's surprising that this button already existed :)

Reply:
This button was added later.